### PR TITLE
Jcs/financial role

### PR DESCRIPTION
**Pull request by jSylvestre**

### CODE REVIEW

**Review by sprucely — APPROVED** [review]
Didn't test, but looks fine.

File: Hippo.Web/Controllers/AdminController.cs [hunk 1]
```diff
@@ -127,6 +127,96 @@ public async Task<IActionResult> RemoveClusterAdmin(int id)
...
+        // get all users with cluster admin permissions
+        return Ok(await _dbContext.Users
+            .AsNoTracking()
+            .Where(u => u.Permissions.Any(p => p.Cluster.Name == Cluster && p.Role.Name == Role.Codes.FinancialAdmin))
+            .OrderBy(u => u.LastName).ThenBy(u => u.FirstName)
+            .ToArrayAsync());
+    }
+
+    [HttpPost]
+    public async Task<IActionResult> AddFinancialAdmin(string id)
```
Comment (srkirkland) [inline Hippo.Web/Controllers/AdminController.cs:143]: seems very repetitive w/ add/remove cluster admin, almost copy/paste could probably be combined

Comment (jSylvestre) [reply Hippo.Web/Controllers/AdminController.cs:143]: Yeah... I got lazy, it is a little different query...

**Review by srkirkland — COMMENTED** [review]
definitely better.  maybe next time just something like `RemoveAdminRole?type=financial` so it can be extended in the future.  we can catch it next time though

**Pull request merged**